Snowflake.Data.Tests/SFDbCommandIT.cs

```diff
@@ -751,22 +752,22 @@ public void TestPutArrayBindAsyncMultiThreading()
 
         static void ThreadProcess1(string connstr)
         {
-            ArrayBindTest(connstr, "testPutArrayBind1", 250000);
+            ArrayBindTest(connstr, "testPutArrayBind1", 15000);

```

Comment:
can you explain to me why change the value from 250,000 to 15,000 and the other changes from 1,250,000 to 25,000?

Reply:
For github Mac vm, usually we got random error when the data size is too large, but on a physical machine it works fine. I just reduce to number of rows to prevent it use up too much vm memory.